src/server/bitops_family.cc

```diff
@@ -310,15 +310,15 @@ class ElementAccess {
 };
 
 std::optional<bool> ElementAccess::Exists(EngineShard* shard) {
-  auto res = shard->db_slice().FindReadOnly(context_, key_, OBJ_STRING);
+  auto res = context_.ns->GetCurrentDbSlice().FindReadOnly(context_, key_, OBJ_STRING);
```

Comment:
Our call chain is already designed to pass shard by argument, therefore it's already in a register. I suggest that you add actually use `GetDbSlice(ShardId)` and avoid doing it implicitly inside namespace. 
Now, it extends the whole expression in so many places, so I also suggest to add a convenience wrapper `ConnectionContext::FindReadOnly`.
why? because we use `context_` twice here, so instead we could shorten this 
in some many places to just `context_.FindReadOnly(key_, OBJ_STRING)`
of course it also applies to other *frequent* operations like FindMutable

Reply:
Wait, sorry, my bad. I thought that your proposal was to make this change in `ConnectionContext`. I now understand you meant in `DbContext`..

Reply:
It looks like the only place which uses DbContext, but not using OpArgs, is bitops family, so I didn't do that.
But I used `shard->shard_id()` as you proposed.

Reply:
Yeah, I confused both types.